src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnectorConfig.java

```diff
@@ -1395,6 +1482,20 @@ public static ConfigDef configDef() {
         return CONFIG_DEFINITION.configDef();
     }
 
+    // Source of the validation rules - https://doxygen.postgresql.org/slot_8c.html#afac399f07320b9adfd2c599cf822aaa3
+    private static int validateReplicationSlotName(Configuration config, Field field, Field.ValidationOutput problems) {
+        final String name = config.getString(field);
+        int errors = 0;
+        if (name != null) {
+            if (!name.matches("[a-z0-9_]{1,63}")) {
+                problems.accept(field, name,
+                        "Valid replication slot name must contain only digits, lowercase characters and underscores with length <= 63");
+                ++errors;
+            }
+        }
+        return errors;
+    }
+
```

Comment:
Was this taken from upstream connector or added by you?

Reply:
This was taken from the upstream connector as a part of declaration of `slot.name` field